src/Illuminate/Support/LazyCollection.php

```diff
@@ -36,6 +36,19 @@ public function __construct($source = null)
             $this->source = $this->getArrayableItems($source);
         }
     }
+    
+    /**
+     * Convert source to iterable
+     *
+     * @param $source
```

Comment:
invalid syntax

Reply:
Still invalid syntax.

Reply:
I am a bit lost. How do I fix this?

Reply:
It's `@param` followed by the **type** and then the variable name `$source`; see any other `@param` in the framework.

Reply:
Okay, I just made a commit that fixed that. Thanks!